src/pages/trails/ballots/components/ballot-form.vue

```diff
@@ -282,8 +300,37 @@ q-dialog(
         color="primary"
         :label="$t('common.buttons.create')"
         :loading="submitting"
-        @click="onAddBallot()"
+        @click="openConfirmation()"
+      )
+    q-dialog(
+      v-model="prompt"
+      persistent
+    )
+      q-card(
+        style="min-width: 350px"
       )
+        q-card-section.row.items-center
+          span.q-ml-sm(
+            v-if="available"
+          ) {{ "There will be a fee for creating a ballot." }}
```

Comment:
Is it possible to change it to `There will be a ${ballotFee} fee for creating a ballot.`?